src/tags/execcc.js

```diff
@@ -46,11 +46,12 @@ module.exports =
                 case 1:
                     return TagManager.list['exec'].execTag(subtag, context, ccommand.content, '');
                 case 2:
-                    return TagManager.list['exec'].execTag(subtag, context, ccommand.content, args[1]);
+                    return TagManager.list['exec'].execTag(subtag, context, ccommand.content, args[1], ccommand.flags);
```

Comment:
Are the flags provided when calling the ccommand possibly contented in the length of `args` ?

Reply:
Is it possible that `args.length` also takes into account the flags? Because it would then change the number of detected arguments then the `case 2:` changes may be useless.

Reply:
Ah, I think I understand your question ow, but I don't quite understand the confusion tbh 🤔 
`args.length` includes the tag/ccommand name and every argument after it. So `b!foo --bar` would have an `args.length` of `2` an be `case 2:` (and thus require `ccommand.flags` to be passed to recognise the `--bar` flag).